catroid/gradle/intellij_config_tasks.gradle

```diff
@@ -21,50 +21,41 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-task gitSkipWorktreeForIntellijConfigFiles << {
-    try {
-        'git update-index --skip-worktree .idea/misc.xml'.execute().text.trim()
-        'git update-index --skip-worktree .idea/encodings.xml'.execute().text.trim()
-        'git update-index --skip-worktree .idea/vcs.xml'.execute().text.trim()
-        'git update-index --skip-worktree Catroid.iml'.execute().text.trim()
-        'git update-index --skip-worktree catroid/src/test/catroidSourceTest.iml'.execute().text.trim()
-        'git update-index --skip-worktree .idea/codeStyleSettings.xml'.execute().text.trim()
-    } catch (IOException exception) {
-        throw new UnsupportedOperationException("Could not find git! Maybe it is not in \$PATH variable?", exception)
-    }
-}
-
-task gitNoSkipWorktreeForIntellijConfigFiles << {
-    try {
-        'git update-index --no-skip-worktree .idea/misc.xml'.execute().text.trim()
-        'git update-index --no-skip-worktree Catroid.iml'.execute().text.trim()
-        'git update-index --no-skip-worktree catroid/src/test/catroidSourceTest.iml'.execute().text.trim()
-        'git update-index --no-skip-worktree .idea/codeStyleSettings.xml'.execute().text.trim()
-    } catch (IOException exception) {
-        throw new UnsupportedOperationException("Could not find git! Maybe it is not in \$PATH variable?", exception)
-    }
-}
+def skipList = ['.idea/misc.xml',
+                '.idea/encodings.xml',
+                '.idea/vcs.xml',
+                'Catroid.iml',
+                'catroid/src/test/catroidSourceTest.iml',
+                '.idea/codeStyleSettings.xml',
+                'catroid/catroid.iml'
+                ]
 
-task gitSaveStashSwitchingToOldBranch << {
-    try {
-        'git update-index --no-skip-worktree .idea/misc.xml'.execute().text.trim()
-        'git update-index --no-skip-worktree Catroid.iml'.execute().text.trim()
-        'git update-index --no-skip-worktree catroid/src/test/catroidSourceTest.iml'.execute().text.trim()
-        'git update-index --no-skip-worktree .idea/codeStyleSettings.xml'.execute().text.trim()
-        'git stash save "IntelliJConfig"'.execute().text.trim()
-    } catch (IOException exception) {
-        throw new UnsupportedOperationException("Could not find git! Maybe it is not in \$PATH variable?", exception)
+task gitSkipWorktreeForIntellijConfigFiles {
+    doLast {
```

Comment:
As written above the leftshift operator is deprecated and will be removed in future releases of Gradle. That is why I replaced it.